SOURCES/tomcat8.conf

```diff
@@ -0,0 +1,48 @@
+# System-wide configuration file for tomcat8 services
+# This will be loaded by systemd as an environment file,
+# so please keep the syntax. For shell expansion support
+# place your custom files as /etc/tomcat8/conf.d/*.conf
+#
+# There are 2 "classes" of startup behavior in this package.
+# The old one, the default service named tomcat8.service.
+# The new named instances are called tomcat8@instance.service.
+#
+# Use this file to change default values for all services.
+# Change the service specific ones to affect only one service.
+# For tomcat8.service it's /etc/sysconfig/tomcat8, for
+# tomcat8@instance it's /etc/sysconfig/tomcat8@instance.
+
+# This variable is used to figure out if config is loaded or not.
+TOMCAT_CFG_LOADED="1"
+
+# In new-style instances, if CATALINA_BASE isn't specified, it will
+# be constructed by joining TOMCATS_BASE and NAME.
+TOMCATS_BASE="/var/lib/tomcat8/"
+
+# Where your tomcat installation lives.
+CATALINA_BASE="/usr/share/tomcat8"
+CATALINA_HOME="/usr/share/tomcat8"
+
+# Where your java installation lives
+JAVA_HOME="/usr/lib/jvm/jre"
+
+# System-wide tmp
+CATALINA_TMPDIR="/var/cache/tomcat8/temp"
+
+# JVM parameters.
+CATALINA_OPTS="-Xms512M -Xmx2048M -server -XX:+UseParallelGC"
+JAVA_OPTS="-Djava.awt.headless=true -Djava.security.egd=file:/dev/./urandom -Djdk.tls.ephemeralDHKeySize=2048"
+
+# You can change your tomcat locale here
+#LANG="en_US"
+
+# Run tomcat under the Java Security Manager
+SECURITY_MANAGER="false"
+
+# Time to wait in seconds, before killing process
+# TODO(stingray): does nothing, fix.
```

Comment:
Just checking to see if this was something you wanted to do before merge.

Reply:
That was "inherited" from the [`tomcat-8.0.conf`](https://src.fedoraproject.org/rpms/tomcat/blob/master/f/tomcat-8.0.conf#_46-48) I imported from Fedora.